api/buildings/admin.py

```diff
@@ -29,6 +30,50 @@ class BuildingAdmin(admin.ModelAdmin):
         "general_id",
     )
     search_fields = ("address",)
+    actions = (
+        "make_choose",
+        "make_accepted",
+        "make_rejected",
```

Comment:
Would change the names from `make_<status>` to `set_status_to_<status>`

Reply:
Ok, I will change that.